main.go

```diff
@@ -2,6 +2,15 @@ package main
 
 import (
 	"go.k6.io/k6/cmd"
+
+	// Blank-importing golang.org/x/crypto/x509roots/fallback bundles a set of
+	// root fallback certificates from Mozilla into the resulting binary. This
+	// allows the program to run in environments where the system root
+	// certificates are not available, for example inside a minimal container.
+	// These are _fallbacks_, meaning that if the system _does have_ a set of
+	// root certificates, those will be given priority. The binary size will
+	// increase a little (~ 220 kB).
+	_ "golang.org/x/crypto/x509roots/fallback"
```

Comment:
I might be wrong, but there could be an issue of having an import in `main.go` any custom binary built with the https://github.com/grafana/xk6 won't have it :cry: 

```sh
$ xk6 build 12a0441084ce27d7235b01fd4aab3a0c7d0987aa
2023/11/07 09:34:47 [INFO] Temporary folder: /tmp/buildenv_2023-11-07-0934.3891399373
2023/11/07 09:34:47 [INFO] Initializing Go module
2023/11/07 09:34:47 [INFO] exec (timeout=10s): /home/olegbespalov/go/bin/go mod init k6
go: creating new go.mod: module k6
2023/11/07 09:34:47 [INFO] Pinning versions
2023/11/07 09:34:47 [INFO] exec (timeout=0s): /home/olegbespalov/go/bin/go mod tidy -compat=1.17
go: warning: "all" matched no packages
2023/11/07 09:34:47 [INFO] Writing main module: /tmp/buildenv_2023-11-07-0934.3891399373/main.go
2023/11/07 09:34:47 [INFO] exec (timeout=0s): /home/olegbespalov/go/bin/go mod edit -require go.k6.io/k6@12a0441084ce27d7235b01fd4aab3a0c7d0987aa
2023/11/07 09:34:47 [INFO] exec (timeout=0s): /home/olegbespalov/go/bin/go mod tidy -compat=1.17
2023/11/07 09:34:47 [INFO] exec (timeout=0s): /home/olegbespalov/go/bin/go mod tidy -compat=1.17
2023/11/07 09:34:47 [INFO] Build environment ready
2023/11/07 09:34:47 [INFO] Building k6
2023/11/07 09:34:47 [INFO] exec (timeout=0s): /home/olegbespalov/go/bin/go mod tidy -compat=1.17
2023/11/07 09:34:47 [INFO] exec (timeout=0s): /home/olegbespalov/go/bin/go build -o /home/olegbespalov/k6 -ldflags=-w -s -trimpath
2023/11/07 09:34:48 [INFO] Build complete: ./k6
2023/11/07 09:34:48 [INFO] Cleaning up temporary folder: /tmp/buildenv_2023-11-07-0934.3891399373

xk6 has now produced a new k6 binary which may be different than the command on your system path!
Be sure to run './k6 run <SCRIPT_NAME>' from the '/home/olegbespalov' directory.
$ go version -m k6 | grep crypto
        dep     github.com/grafana/xk6-webcrypto        v0.1.0  h1:StrQZkUi4vo3bAMmBUHvIQ8P+zNKCH3AwN22TZdDwHs=
        dep     golang.org/x/crypto     v0.14.0 h1:wBqGXzWJW6m1XrIKlAH0Hs1JJ7+9KBwnIO8v66Q9cHc=
```

Reply:
I moved the import to "cmd"